src/index.ts

```diff
@@ -1,13 +1,11 @@
 export interface ImplementMe {
```

Comment:
It's a small thing, but can we have this in `lib/` instead of `src`? It's for a library after all. I don't feel too strongly but it seems to make more sense to me.

Reply:
I was actually checking a few up-to-date libraries to figure which folder we should be using.
`pinejs-client-js` is now using `src` for example and it's mainly the SDK which still uses a `lib` folder. Imo, since the code needs to be compiled and isn't consumable right away, it makes some sense to use `src` as the folder. Let me know what you think.

Reply:
My understanding has always been libraries have their source in `lib`, but apps in `src` - for example if a library *also* had a cli interface, it would have both a `src` and `lib`

Reply:
@CameronDiver I did some research on this and
* I couldn't find an official statement about this from nodejs
* Most discussions suggest that `lib` is meant to hold "ready to consume with `require()`" code which doesn't need compilation/conversion
See: https://www.reddit.com/r/learnprogramming/comments/6si2cq/confused_about_lib_vs_src_in_npm_module_package/
See: https://gist.github.com/tracker1/59f2c13044315f88bee9#lib--src
See: https://stackoverflow.com/questions/38426078/why-use-lib-vs-src-directory-names-in-javascript-which-is-standard

We do have a few libraries which use `src:
See: https://github.com/balena-io/pinejs
See: https://github.com/balena-io-modules/pinejs-client-js
See: https://github.com/balena-io-modules/odata-to-abstract-sql
See: https://github.com/balena-io-modules/abstract-sql-compiler
See: https://github.com/balena-io-modules/sbvr-types
See: https://github.com/balena-io-modules/rendition
But we also have others that use `lib`
See: https://github.com/balena-io/balena-sdk/
See: https://github.com/balena-io-modules/balena-auth
See: https://github.com/balena-io-modules/node-balena-lint

I will change this back to `lib` and we can change that later if we want.

Reply:
sorry yeah I meant my understanding of how we use it in balena. I think we don't have a standard, given your research. Honestly with this, I'm happy for it to be `src/`, although we send the ready-to-consume files to `build/` not `lib/`.